### PR TITLE
create login and structure for Audiomack website

### PR DESCRIPTION
**Pull request by Cxristian**

### CODE REVIEW

File: src/main/java/audiomack/ui/pages/LoginPage.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+/**
+ * LoginPage class
+ *
+ * @author Cristian Lujan
+ * @version 0.0.1
+ */
+public class LoginPage extends BasePage {
+
+    @FindBy(id = "react-view")
+    WebElement audiomackForm;
```
Comment (alszla) [inline src/main/java/audiomack/ui/pages/LoginPage.java]: Shouldn't an access modifier be specified for this attribute(s)?

File: src/main/java/soundCloud/PageTransporter.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
+/*
```
Comment (alszla) [inline src/main/java/soundCloud/PageTransporter.java]: I guess the "soundcloud" directory is not part of your testing project anymore.

File: src/main/java/soundCloud/ui/pages/LoginPage.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
+/*
```
Comment (alszla) [inline src/main/java/soundCloud/ui/pages/LoginPage.java]: I guess the "soundcloud" directory is not part of your testing project anymore.

File: src/main/java/soundCloud/ui/BasePage.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
+/*
```
Comment (alszla) [inline src/main/java/soundCloud/ui/BasePage.java]: I guess the "soundcloud" directory is not part of your testing project anymore.

File: src/main/java/soundCloud/PageTransporter.java (hunk 2 above)
Comment (Cxristian) [reply src/main/java/soundCloud/PageTransporter.java]: OMG...
xd

Comment (Cxristian) [reply src/main/java/soundCloud/PageTransporter.java]: done

File: src/main/java/audiomack/ui/pages/LoginPage.java (hunk 1 above)
Comment (Cxristian) [reply src/main/java/audiomack/ui/pages/LoginPage.java]: done

File: src/main/java/soundCloud/ui/pages/LoginPage.java (hunk 3 above)
Comment (Cxristian) [reply src/main/java/soundCloud/ui/pages/LoginPage.java]: done

**Review by alszla — APPROVED** [review]

**Pull request merged**